app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <tm-page data-title="Validator">
-    <tm-data-loading v-if="delegates.loading" />
```

Comment:
why?

Reply:
this displayed the loader on disconnection on top of the other component, which was wrong